src/controllers/expenses.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const expensesService = require('../services/expenses.js');
+const usersService = require('../services/users.js');
+
+const getAll = async(req, res) => {
```

Comment:
Use destructuring assignment to obtain 'query' from 'req'. Example: const { query } = req;